libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/InfoPanelHeaderComponents.kt

```diff
@@ -0,0 +1,26 @@
+package com.mapbox.navigation.dropin.internal
```

Comment:
Can we rename this to `NavigationViewContextEx`? At the moment the only extensions on top of `NavigationViewContext` is related to `InfoPanelHeader` and hence it makes sense for the choice of naming convention. However, extensions are not limited to just these and we can have more in upcoming improvements/fixes. It would be good to have the name attributed to extensions on top of `NavigationViewContext` rather than associating it with only `InfoPanelHeader`.